api/cmd/api/main.go

```diff
@@ -81,9 +82,18 @@ func main() {
 	characterInfoRepository := repository.NewCharacterInfoDbImpl(db)
 	storyInfoV2Repository := repository.NewStoryInfoV2DbImpl(db)
 
-	storyContentRepository, err := repository.NewStoryContentS3Impl(s3Client, cfg.S3ContentBucketName)
-	if err != nil {
-		logger.Fatalf("Failed to init story content s3 implementation: %v", err)
+	var storyContentRepository repository.StoryContentRepository
+	var storyError error
+
+	if xconfig.GetEnv() == xconfig.Environments.Local {
+		fmt.Println("local reached")
+		storyContentRepository, storyError = repository.NewStoryContentFsImpl(cfg.ContentPath)
+	} else {
+		fmt.Println(xconfig.GetEnv())
+		storyContentRepository, storyError = repository.NewStoryContentS3Impl(s3Client, cfg.S3ContentBucketName)
+		if storyError != nil {
```

Comment:
This error check should by outside of the `else` block to check error for both condition. Otherwise the error in `if` is not checked